packages/zod-openapi/test/index.test.ts

```diff
@@ -1779,21 +1779,21 @@ describe('RouteConfigToTypedResponse', () => {
             age: number
           },
           200,
-          'json' | 'text'
+          'json'
```

Comment:
Since now the `TypedResponse` only allows `json` with `application/json`, I fixed the test.